SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -73,20 +73,22 @@ Worker_RequestId USpatialSender::CreateEntity(USpatialActorChannel* Channel)
 	FString ClientWorkerAttribute = GetOwnerWorkerAttribute(Actor);
 
 	WorkerAttributeSet ServerAttribute = { SpatialConstants::ServerWorkerType };
+	WorkerAttributeSet AIWorkerAttribute = { FString(TEXT("AIWorker")) };
 	WorkerAttributeSet ClientAttribute = { SpatialConstants::ClientWorkerType };
 	WorkerAttributeSet OwningClientAttribute = { ClientWorkerAttribute };
 
 	WorkerRequirementSet ServersOnly = { ServerAttribute };
+	WorkerRequirementSet AllServerTypes = { ServerAttribute, AIWorkerAttribute };
 	WorkerRequirementSet ClientsOnly = { ClientAttribute };
 	WorkerRequirementSet OwningClientOnly = { OwningClientAttribute };
 
-	WorkerRequirementSet AnyUnrealServerOrClient = { ServerAttribute, ClientAttribute };
-	WorkerRequirementSet AnyUnrealServerOrOwningClient = { ServerAttribute, OwningClientAttribute };
+	WorkerRequirementSet AnyUnrealServerOrClient = { ServerAttribute, AIWorkerAttribute, ClientAttribute };
+	WorkerRequirementSet AnyUnrealServerOrOwningClient = { ServerAttribute, AIWorkerAttribute, OwningClientAttribute };
 
 	WorkerRequirementSet ReadAcl;
 	if (Class->HasAnySpatialClassFlags(SPATIALCLASS_ServerOnly))
```

Comment:
This flag name does not seem to be matching with the new value

Reply:
Yes, I'm currently not sure what we prefer to do here. Will follow up with the GDK team

Reply:
what's the outcome of that conversation?